sydent/http/httpclient.py

```diff
@@ -57,3 +79,21 @@ def post_json_get_nothing(self, uri, post_json, opts):
         )
         defer.returnValue(response)
 
+@implementer(IPolicyForHTTPS)
+class FederationPolicyForHTTPS(object):
+    def creatorForNetloc(self, hostname, port):
+        context = SSL.Context(SSL.SSLv23_METHOD)
+        try:
+            _ecCurve = _OpenSSLECCurve(_defaultCurveName)
```

Comment:
can you do matrix-org/synapse#3157 to this please

Reply:
done